tox.ini

```
@@ -1,9 +1,10 @@
 [tox]
-envlist = py{37,38,39}-django{111,22,32},py310-django{22,32}
+envlist = py{37,38,39}-django{111,22,32,40},py310-django{22,32,40}
```

Ah, when I manually selected all of the py37 environments to run locally (`tox -e py37-django111,py37-django22,py37-django32,py37-django40`) I learned that Django 4.x will not install there -- [only Python 3.8, 3.9, 3.10 supported](https://docs.djangoproject.com/en/4.0/releases/4.0/#python-compatibility)

Maybe it's this? 👇 

```suggestion
envlist = py37-django{111,22,32}, py{38,39}-django{111,22,32,40}, py310-django{22,32,40}
```